spec/features/admin/working_days_spec.rb

```diff
@@ -100,14 +109,21 @@ def working_days_setting
 
     expect(working_days_setting).to eq([2, 3, 4])
 
-    expect(work_package.reload.start_date)
-      .to eql(Date.parse('2022-09-20'))
+    expect_schedule(WorkPackage.all, <<~CHART)
+      days                  | MTWTFSSmtwtfssmtwt  |
+      earliest_work_package |  XXX....XX          |
+      second_work_package   |    X....XXX         |
+      follower              |                XXX  |
+    CHART
 
-    expect(work_package.due_date)
-      .to eql(Date.parse('2022-09-28'))
+    # The updated work packages will have a journal entry informing about the change
+    wp_page = Pages::FullWorkPackage.new(earliest_work_package)
+    wp_page.visit!
+
+    wp_page.expect_comment(text: "Working days changed (Monday is now non-working, Friday is now non-working).")
   end
 
-  it 'shows error when non working days are set' do
+  it 'shows error when non working days are all unset' do
```

Comment:
This one is funny, because you changed it from `'shows error when no working days are set'` to `'shows error when non working days are set'` in another PR review, and now you're fixing it again because the new text did not match the test content anymore.

Sorry, I should have reviewed it on the first time you changed it.

Reply:
In either case, good that it good fixed.